### DISCUSSION TITLE
Always use vertical column layout

### DISCUSSION THREAD

**Discussion by ldelossa** [jesseduffield/lazygit#3036] [Q&A]
Is it possible to have lazygit always use a vertical layout, like it does when the terminal gets small? I enjoy this layout since it allows the most mount of text within each view to be displayed. I tried the 'mainPanelSplitMode' config set to vertical but I don't believe this is what its used for. 

Thanks

**Comment by stefanhaller** [comment]
`mainPanelSplitMode: vertical` causes the unstaged and staged views to go on top of each other rather than side by side. If this is not what you want to achieve, then what else is it?

**Reply by ldelossa** [reply]
I want all views to be stacked ontop of each other. The same layout that occurs when the terminal width is less then 80 chars I believe. But i want this layout always, not only when the terminal is less then 80 chars

**Reply by stefanhaller** [reply]
Oh I see. I have never seen that mode; fascinating.

It's called "portraitMode" [in the code](https://github.com/jesseduffield/lazygit/blob/cdb1d76484d9c10d5bf7cd10e5c9d597856591f2/pkg/gui/controllers/helpers/window_arrangement_helper.go#L43), and I suppose it would be pretty easy to add a config to turn it on always. Are you interested in giving it a try?

**Reply by ldelossa** [reply]
Sure: https://github.com/jesseduffield/lazygit/pull/3037
